tests/QnA-testData.js

```diff
@@ -1,159 +1,266 @@
+  //       question_date:"2018-01-04T00:00:00.000Z",
+  //       asker_name:"yankeelover",
+  //       question_helpfulness:1,
+  //       reported:false,
+  //       answers:
+  //         { 1992356:
+  //             { id:1992356,
+  //               body:"Something pretty soft but I can't be sure",
+  //               date:"2018-01-04T00:00:00.000Z",
+  //               answerer_name:"metslover",
+  //               helpfulness:5,
+  //               photos:
+  //               [
+  //                 "https://images.unsplash.com/photo-1530519729491-aea5b51d1ee1?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1651&q=80","https://images.unsplash.com/photo-1511127088257-53ccfcc769fa?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1650&q=80","https://images.unsplash.com/photo-1500603720222-eb7a1f997356?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1653&q=80"
+  //               ]},
+  //           1992358:
+  //             { id:1992358,
+  //               body:"Its the best! Seriously magic fabric",
+  //               date:"2018-01-04T00:00:00.000Z",
+  //               answerer_name:"metslover",
+  //               helpfulness:7,
+  //               photos:[]
+  //             },
+  //           1992359:
+  //             { id:1992359,
+  //               body:"DONT BUY IT! It's bad for the environment",
+  //               date:"2018-01-04T00:00:00.000Z",
+  //               answerer_name:"metslover",
+  //               helpfulness:8,"photos":[]
+  //             },
+  //           1992408:
+  //             { id:1992408,
+  //               body:"Suede",
+  //               date:"2018-11-04T00:00:00.000Z",
+  //               answerer_name:"metslover",
+  //               helpfulness:7,
+  //               photos:[]
+  //             },
+  //           1992446:
+  //             { id:1992446,
+  //               body:"Supposedly suede, but I think its synthetic",
+  //               date:"2018-12-04T00:00:00.000Z",
+  //               answerer_name:"metslover",
+  //               helpfulness:3,
+  //               photos:[]}}
+  //             },
+
+
+  //   ]
+  // },
+ { product_id:"28213",
+   results:
+     [
+       { question_id:213343,
+         question_body:"Where does this product ship from?",
+         question_date:"2018-01-06T00:00:00.000Z",
+         asker_name:"jbilas",
+         question_helpfulness:36,
+         reported:false,
+         answers:
+           { 1992374:
+              { id:1992374,
+               body:"It ships from the facility in Tulsa",
+               date:"2018-01-06T00:00:00.000Z",
+               answerer_name:"dschulman",
+               helpfulness:30,
+               photos:[]},
+             2171318:
+              { id:2171318,
+                body:"This is my test answer",
+                date:"2021-07-23T00:00:00.000Z",
+                answerer_name:"testUsername123",
+                helpfulness:0,
                 photos:
```

Comment:
Break between what is added and what is removed a little unclear in the view in GH. If it all looks right in editor no worries, just looks a bit like the data is spliced in strange places here